### PR TITLE
Add criteria CR120 through CR122 and list under CM55

### PR DESCRIPTION
**Pull request by kristovatlas**
Added:
- CR120
  (8732471400c61f1bdedb5c2d065635bc43b3b7e93d1295b9263794f818609758)
- CR121
  (bb69cef4049d52d46a9da90e3daeceebee83768d87cb0d0f7d92e5e9bffcdab2)
- CR122
  (f09d7dcb1f36adec20de3b0e60b5c6d27bd6ac64547b5a9ce67f2858b3e55cdf)

Listed under CM55
(fbbb0c4c77661dbdd2da52225e8a9aa42aa784699062c25b1161ac8f4a9c7c2f)

This is according to our planned changes discussed in
https://github.com/OpenBitcoinPrivacyProject/wallet-ratings/issues/60

### CODE REVIEW

File: obpp3.json [hunk 1]
```diff
@@ -1498,6 +1498,24 @@
                                             "id": "OBPPV3/CR71",
                                             "description": "A user can produce a compiled version of the application from the public source code that exactly matches the version distributed by the wallet provider",
                                             "effectiveness": 0
+                                        },
+                                        {
+                                            "numeral": "b",
+                                            "id": "OBPPV3/CR120",
+                                            "description": "Before applying a software update, the client notifies the user of the udpate and provides an opportunity to intervene before wallet private keys or meta data are decrypted using udpated software",
```
Comment (kristovatlas) [inline obpp3.json]: s/udpated/updated

**Comment by justusranvier** [comment]
ACK

**Comment by crwatkins** [comment]
ACK

**Comment by kristovatlas** [comment]
merge conflicts resolved. this was previously ACK'd

**Pull request merged**